src/main/java/com/spikes2212/command/genericsubsystem/commands/MoveGenericSubsystemWithPID.java

```diff
@@ -14,52 +16,65 @@
  * @see GenericSubsystem
  */
 public class MoveGenericSubsystemWithPID extends CommandBase {
-
     /**
      * the subsystem the command moves.
      */
     private final GenericSubsystem subsystem;
 
     /**
-     * the pid loop that calculates the speed for the command
+     * A supplier that returns the subsystem's current location.
+     */
+    private Supplier<Double> source;
+
+    /**
+     * The PID Settings for the PID control loop.
      */
-    private final PIDLoop pidLoop;
+    private PIDSettings pidSettings;
```

Comment:
please add a `FeedFarwardController` instance to this command and adapt the constructor accordingly.